src/Chapter1.hs

```diff
@@ -508,7 +521,9 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y =
+    if abs x < abs y then x
+    else y
```

Comment:
Usually, by most Haskell style guides, you either put `if-then-else` on a single line, or put `then` and `else` on separate lines for readability. Something like:

```haskell
if abs x < abs y then x else y
```

or

```haskell
if abs x < abs y
then x
else y
```

Just for readability :slightly_smiling_face: Don't worry, nothing to fix here, just a minor stylistic suggestion